string-manipulation-3/src/is-palindromic.js

```diff
@@ -0,0 +1,15 @@
+/* exported isPalindromic */
+// define function
+function isPalindromic(string) {
+  // storage for list that contains string without any spaces
+  var stringArr = string.split(' ').join('').split('');
+  // storage for list that contains reversed string without any spaces
+  var stringArrReverse = string.split(' ').join('').split('').reverse();
+  // if string is equal to reversed string,
+  if (stringArr.join('') === stringArrReverse.join('')) {
```

Comment:
2 more loops 👀

Reply:
Try and redo this problem with fewer loops necessary. You can get it down to only two or three loops total.